ha/k8s_setup/ha_setup.py

```diff
@@ -199,9 +219,21 @@ def process(self):
             with open(const.HA_CONFIG_FILE, 'w+') as conf_file:
                 yaml.dump(conf_file_dict, conf_file, default_flow_style=False)
 
+            Cmd.copy_file(const.SOURCE_HEALTH_HIERARCHY_FILE, const.HEALTH_HIERARCHY_FILE)
             # First populate the ha.conf and then do init. Because, in the init, this file will
             # be stored in the confstore as key values
             ConfigManager.init("ha_setup")
+
+            machine_id = self.get_machine_id()
+            cluster_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}cluster_id')
+            # site_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}site_id')
+            site_id = '1'
+            # rack_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}rack_id')
+            rack_id = '1'
+            conf_file_dict.update({'COMMON_CONFIG': {'cluster_id': cluster_id, 'rack_id': rack_id, 'site_id': site_id}})
```

Comment:
How will this get updated into consul ?

Reply:
Just below lines have code to dump this data in yaml format to ha.conf

Reply:
yes, but the configManager init was going to pickup the ha.conf data and put it in confstore right ? where will that happen ?

Reply:
As discussed, it will be available in the confstore after starting one of the service. But just added TODO here to make sure everything is in place